tests/test_tracker.py

```diff
@@ -73,7 +73,7 @@ def test_tracker_interactions(per_run_unique):
 def test_tracker_create_list():
     """Tests that we can create a list of trackers in bulk."""
     trackers_list = easypost.Tracker.create_list(
-        trackers={
+        params={
```

Comment:
The goal of this PR is to avoid the need to pass in a field at all, is that not possible with this implementation?

Reply:
We need a map to store those maps.